docs/using-the-compiler.rst

```diff
@@ -9,6 +9,8 @@ Using the compiler
 Using the Commandline Compiler
 ******************************
 
+If you installed Solidity using npm then run the compiler with ``solcjs``.
```

Comment:
Actually most of the features described here are not supported by `solcjs`. I'd rather just include a `.. note ::` mentioning that.

Reply:
The inconsistency is painful. If npm is the recommended install method then the main usage instructions should target an npm install. The additional instructions should be a separate section, because the reader will need to stop what they're doing, go back and then reinstall everything from the not default install instructions.